src/test/javascript/spec/helpers/mocks/service/mock-server-date.service.ts

```diff
@@ -0,0 +1,18 @@
+import { ServerDateService } from 'app/shared/server-date.service';
+import moment from 'moment';
```

Comment:
did not we decide not using "moment" anymore after last client update? or am I missing sth?

Reply:
No, had quite some merge conflicts. Must have missed that one.